### PR TITLE
Update DG

### PR DESCRIPTION
**Pull request by EksdeeeX**
updated DG

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -449,13 +466,64 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case resumes at step 3.
 
+* 3c. User enters an invalid contact information
+   * 3c1. System displays that the index is invalid and to try again.
+
+      Use case resumts at step 3.
```
Comment (glozxi) [inline docs/DeveloperGuide.md:472]: Small typo here.

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -449,13 +466,64 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case resumes at step 3.
 
+* 3c. User enters an invalid contact information
+   * 3c1. System displays that the index is invalid and to try again.
```
Comment (glozxi) [inline docs/DeveloperGuide.md:470]: Should "index" be changed to "contact information"?

**Review by glozxi — CHANGES_REQUESTED** [review]
Looks good overall, just a few nits.

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (EksdeeeX) [reply docs/DeveloperGuide.md:472]: Fixed typo!

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (EksdeeeX) [reply docs/DeveloperGuide.md]: Changed to "contact information", good catch!

**Review by glozxi — APPROVED** [review]
LGTM

**Pull request merged**